src/boot/index.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+// This is the main entry point for the Hypothesis client in the host page
+// and the sidebar application.
+
+/* global __MANIFEST__ */
+
+var boot = require('./boot');
+
+// The HYP* properties are set (if at all) by the service which is serving the
+// Hypothesis client to tell it where to load the sidebar and assets from.
+var appHtmlUrl = window.__HYP_APP_HTML_URL__ || 'https://hypothes.is/app.html';
```

Comment:
Hmm, yes I think they could be. These values are written by the tiny script that the service's /embed.js route serves up. That script could equally well create a settings script tag.